README.md

```diff
@@ -1,28 +1,65 @@
 # XMTP-MEMO-JS
 
-**XMTP client SDK for Pre-Registration messaging**
+![Status](https://img.shields.io/badge/Project_Status-Developer_Preview-red)
 
-`xmtp-memo-js` provides a TypeScript implementation of an XMTP memo client.
-Build with `xmtp-memo-js` to send 'messages' to a blockchain wallet address which does not have published encryption keys on the network. `xmtp-memo-js` supports use-cases where content needs to be sent to a account which has not registered yet.
+## XMTP client SDK for pre-registration messaging
 
-**What is a memo?**
+`xmtp-memo-js` provides a TypeScript implementation of an XMTP memo client.
 
-Memo's are similar to XMTP Messages however they differ substantively.
+With XMTP, a blockchain account address must be registered on the XMTP network to send and receive end-to-end encrypted messages. Build your app with `xmtp-memo-js` to enable a registered address to start communicating with an unregistered address by sending a memo instead of a message.
```

Comment:
Per feedback from Rich, called out the baseline requirement that an address must be registered to send and receive messages.

Per feedback from Yash, made an adjustment to the text to make the distinction between memos and messages early on - with your great "What is a memo?" section following up with the deets!